server/players.py

```diff
@@ -76,6 +79,39 @@ def __init__(
         self._game = lambda: None
         self._game_connection = lambda: None
 
+    @property
+    def global_rating(self):
+        return self._global_rating
+
+    @global_rating.setter
+    def global_rating(self, value: Rating):
```

Comment:
wrong typing, use `Union[Tuple[int, int], Rating]` https://docs.python.org/3/library/typing.html#typing.Union

Reply:
This is copied from BasePlayer, so I didn't bother touching it. Frankly you know my opinion on type hints, in trivial cases like this one they bring exactly zero benefit.